scripts/upload-to-aws-bucket.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
+const AWS = require('aws-sdk');
+const request = require('request');
+const db = require('../database/index');
+
```

Comment:
Cool stuff to learn, all of this is news to me.

Reply:
It was a bit of a challenge understanding their documents and how I really am supposed to save on S3, but it was pretty interesting in the end.